R/get_pvol_de.R

```diff
@@ -98,6 +100,9 @@ list_to_pvol <- function(x, time, radar,
   output$scans <- x
 
   output$attributes <- x[[1]]$attributes
+  output$attributes$what[c('starttime','startdate','endtime','enddate')]<-NULL
+  output$attributes$what$date<-sort(unlist(lapply(lapply(lapply(x,'[[','attributes'),'[[', 'what'),'[', c('startdate'))))[1]
+  output$attributes$what$time<-sort(unlist(lapply(lapply(lapply(x,'[[','attributes'),'[[', 'what'),'[', c('starttime'))))[1]
   output$attributes$what$object <- "PVOL"
   output$attributes$what$source <- source
   output$geo <- attr(x[[1]]$params[[1]], "geo")
```

Comment:
```suggestion
   output$attributes <- purrr::chuck(x, 1, "attributes")
  output$attributes$what[c("starttime", "startdate", "endtime", "enddate")] <- NULL
  output$attributes$what$date <- min(purrr::map_chr(x, ~ purrr::chuck(.x, "attributes", "what", "startdate")))
  output$attributes$what$time <- min(purrr::map_chr(x, ~ purrr::chuck(.x, "attributes", "what", "starttime")))
  output$attributes$what$object <- "PVOL"
  output$attributes$what$source <- source
  output$geo <- attr(purrr::chuck(x, 1, "params", 1), "geo")
```

Just a small suggestion, since you are using purrr anyway higher up, we could use purrr here again. I think this is more readable, but that might be personal bias. But should be more defensive in case an element like `what` or `attributes` is missing, base R will silently return `NULL` but `purrr::chuck()` will fail. 

We could consider parsing the datetime and starttime, but this works just fine I think. This `min()` approach on a character vector is just shorthand for `\(x) head(sort(x), 1)`.